bindings/python/crocoddyl/utils/biped.py

```diff
@@ -191,15 +191,20 @@ def createSwingFootModel(self, timeStep, supportFootIds, comTask=None, swingFoot
             comTrack = crocoddyl.CostModelCoMPosition(self.state, comTask, self.actuation.nu)
             costModel.addCost("comTrack", comTrack, 1e6)
         for i in supportFootIds:
-            cone = crocoddyl.FrictionCone(self.nsurf, self.mu, 4, False)
-            frictionCone = crocoddyl.CostModelContactFrictionCone(
+            cone = crocoddyl.WrenchCone(np.identity(3), self.mu, np.array([0.1, 0.05]))
+            wrenchCone = crocoddyl.CostModelContactWrenchCone(
                 self.state, crocoddyl.ActivationModelQuadraticBarrier(crocoddyl.ActivationBounds(cone.lb, cone.ub)),
-                crocoddyl.FrameFrictionCone(i, cone), self.actuation.nu)
-            costModel.addCost(self.rmodel.frames[i].name + "_frictionCone", frictionCone, 1e1)
+                crocoddyl.FrameWrenchCone(i, cone), self.actuation.nu)
+            costModel.addCost(self.rmodel.frames[i].name + "_wrenchCone", wrenchCone, 1e1)
+            # cone = crocoddyl.FrictionCone(self.nsurf, self.mu, 4, False)
```

Comment:
We do not need to keep the commented lines, go ahead and remove them.

Due to this change in the example, I need to push a  modification of a log file. Unfortunately, I need to do it myself. I just want to let you know this.